json/de.json

```diff
@@ -37,7 +37,7 @@
     "Please confirm your password before continuing.": "Bitte bestätigen Sie Ihr Passwort bevor Sie fortfahren.",
     "Regards": "Mit freundlichen Grüßen",
     "Register": "Registrieren",
-    "Remember Me": "Angemeldet bleiben",
+    "Remember me": "Angemeldet bleiben",
```

Comment:
I suggest you keep both keys so it seems repeated

Reply:
Yes, you are right.